quest/util/param_util.py

```diff
@@ -28,8 +24,12 @@ def __init__(self, filters=None, queries=None, **params):
 
     @property
     def datasets(self):
-        datasets = quest.api.get_datasets(filters=self.filters, queries=self.queries, expand=True)
-        return [NamedString(v['display_name'], k) for k, v in datasets.items()]
+        datasets = quest.api.get_datasets(
+            filters=self.filters,
+            queries=self.queries,
+            expand=True
+        )
+        return [NamedString(k, v['display_name']) for k, v in datasets.items()]
```

Comment:
Can you explain why the key is in front now rather before the value? Just so we have something to reference to. Thanks

Reply:
This is consistent with how it was designed to be, and how it needs to be for the Quest App. The named string accepts a `name` and a `string`. This is used to populate a select dropdown widget that needs a string to display and a value associated with it. The named string object provides that where the `name` becomes the `value` and the `string` is what should be displayed to the end user in the widget. This is how it was originally. It got switched on Jun 1st for some reason, but I don't remember why, but this is how it needs to be to work correctly with the app.

Reply:
@AaronV77 pointed out that we changed it in June to work with ParamNB, which assumes that the `get_range` call of an `ObjectSelector` param object will return (`name`, `object`) tuples. The Quest app assumes that the tuples are (`value`, `name`), where the `value` in this case would be the `object`. I've changed the `NamedString` constructor order to be consistent with the way this PR is passing in the arguments, which is what makes the most sense to me right now.